paper/paper.md

```diff
@@ -42,13 +42,13 @@ bibliography: paper.bib
-VILLASnode was originally designed for co-simulation of electrical networks, but was developed to bigger variety of use cases and domains.
-Whereas distributed computing, systems or algortihms aim to solve a common task, geographically distributed experiments link infrastructures with different components to make these components accessible to other infrastructures. Thus, data exchanges are possible which would not be possible in one single infrastructure.
-VILLASnode serves as the gateway that connects components across different infrastructures by providing a set of protocols and customized third-party implementations, e.g., different simulators. It enables seamless collaboration in research and testing environments while safeguarding the intellectual property of the infrastructures. The components at every infrastructure appear as a black box. The infrastructure does not need to share models or confidential information.
+VILLASnode is a multi-protocol gateway, designed to facilitate real-time data exchange between various components of geographically distributed real-time experiments. Components can be testbeds, digital real-time simulators, software tools, and physical devices. It is designed for the co-simulation of power system and related energy applications.
+VILLASnode was originally designed for co-simulation of electrical networks, but was further developed for a bigger variety of use cases and domains.
+Whereas distributed computing, systems, and algortihms aim to solve a common task, geographically distributed experiments link infrastructures with different components to make these components accessible to other infrastructures. Thus, data exchanges are possible that would not be possible in one single infrastructure.
+VILLASnode serves as the gateway that connects components across different infrastructures by providing a set of protocols and customized third-party implementations, e.g., different simulators. It enables seamless collaboration in research and testing environments while safeguarding the intellectual property of the infrastructures. The components of every infrastructure appear as a black box. The infrastructure does not need to share models or confidential information.
 
-VILLASnode is a Linux command line tool and can run as installation from source or as container. It is written in C/C++ and designed in a modular and extensible way.
-All components which are interfaced by the VILLASnode gateway are represented by nodes ($n$). These nodes act as sinks or sources for data specific to the component. Every node is an instance of a node-type. In a single VILLASnode instance, multiple instances of the same node-type can co-exist at the same time.
+VILLASnode is a Linux command line tool, which can run as installed from source or as a container. It is written in C/C++ and designed in a modular and extensible way.
```

Comment:
```suggestion
VILLASnode is a library, written in C/C++ and designed in a modular and extensible way.
```
VILLASnode is not a command line tool. `villas-node` is a command line tool, but this is only one of the binaries build by VILLASnode